tests/unit/test_reward_aggregation.py

```diff
@@ -84,7 +66,7 @@ def test_aggregate_orderbook_rewards(self):
             {
                 "receiver": ["0x1", "0x2", "0x3", "0x4"],
                 "num_trades": [3, 3, 2, 3],
-                "amount": [to_wei(87), to_wei(100), to_wei(107), to_wei(74)],
+                "amount": [to_wei(87), to_wei(97), to_wei(107), to_wei(74)],
```

Comment:
@fleupold this test already identifies a transaction hash where you can see what the difference would be. Is this not what you were looking for?

What was previously a reward of 40 for an order gets projected back to 37. The transaction hash is in the test above the changed code lines. 

Furthermore, there do not appear to be any batches last week which would fall under this reward mapping. As indicated by [this dune query](https://dune.com/queries/1749029)

Reply:
https://etherscan.io/tx/0x43bfe76d590966c7539f1ea0bb7989edc1289f989eaf8d84589c3508c5066c2c for convenience.

Ok, this is indeed a risk free batch (user order settling against PLM crafted liquidity), so this is a good sanity check.